charger/openwb/native/rfid.go

```diff
@@ -0,0 +1,103 @@
+package native
+
+import (
+	"context"
+	"fmt"
+	"slices"
+	"strings"
+	"sync"
+
+	"github.com/evcc-io/evcc/util"
+	"github.com/holoplot/go-evdev"
+)
+
+// NewRFIDHandler initializes RFID device monitoring and returns the channel for RFID reads.
+// It also returns a cancel function to stop monitoring and clean up resources.
+func NewRFIDHandler(ctx context.Context, log *util.Logger) (chan string, func(), error) {
+	devicePaths, err := evdev.ListDevicePaths()
+	if err != nil {
+		return nil, nil, fmt.Errorf("cannot list device paths: %w", err)
+	}
+
+	var keyboardPaths []string
+	for _, d := range devicePaths {
+		log.INFO.Printf("Device path: %s | Name: %s\n", d.Path, d.Name)
+		dev, err := evdev.Open(d.Path)
+		if err != nil {
+			log.INFO.Printf("Cannot read %s: %v\n", d.Path, err)
```

Comment:
Bitte keine Warnungen in INFO Loggen. Was soll der Anwender damit anfangen? WARN oder DEBUG?